addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/handler/SamsungTvHandler.java

```diff
@@ -11,13 +11,6 @@
 import static org.openhab.binding.samsungtv.SamsungTvBindingConstants.POWER;
 import static org.openhab.binding.samsungtv.config.SamsungTvConfiguration.HOST_NAME;
 
-import java.util.ArrayList;
```

Comment:
I'll fix this imports from Eclipse before merging it and while processing any other comments.